package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zenko/cloudserver",
-  "version": "8.7.20",
+  "version": "8.7.21",
```

Comment:
8.7.22 has been released already : please rebase & do the "expected" bump

(it is better to only add this kind of release bump at the last minute, once the PR has been approved)

Reply:
Ready to re-review, will bump and resolve conflicts when PR is approved